package.json

```diff
@@ -99,11 +93,17 @@
     "ember-cli-path-utils": "^1.0.0",
     "ember-cli-sass": "5.6.0",
     "ember-cli-version-checker": "~2.0.0",
+    "ember-computed-decorators": "~0.3.0",
+    "ember-concurrency": "~0.7.19",
+    "ember-elsewhere": "1.0.1",
```

Comment:
Just curious is there anything blocks us from floating ember-elsewhere?

Reply:
The focus of this effort right now is to work through the mess of dependencies being managed as dependencies, devDependencies, and/or being installed via the Ember CLI add-on installation process, it is not to change versions of dependencies, make sure that the new ones are compatible, etc.  That will be a separate effort.